javascripts/landing.js

```diff
@@ -83,4 +83,19 @@ window.addEventListener('DOMContentLoaded', (event) => {
     $articleList.find('li:last-of-type a').click();
     params.delete('last');
   }
+
+  $blog.find("div.subscribe").on('click', function(e) {
```

Comment:
@MeghnaAJ Please move this into a seperate PR

Reply:
Sure, I'll create a separate PR for subscribe once the blog / blog post PRs are merged.